### ISSUE TITLE
Statistics in the decisions tab of the Campaings section not working properly

### ISSUE THREAD

**Issue by avara** [mautic/mautic#13807]
### Mautic Version

4.4.x series

### Way of installing

Other (please specify if relevant in the bug report)

### PHP version

8.0.20 & 8.0.30

### What browsers are you seeing the problem on?

Not relevant

### What happened?

When we send a campaign  to a segment with Y number of contacts and with a decision of "Opens email" to trigger opened emails and get all have been sent and as read X number of contacts, when go to Campaings section and click in Decision tab the statistics of each decision shows a wrong data in the pending actions (always zero) and success percent (always 100%).

We checked the issue is since 4.4.1 version and until last version of 5.x series. We located the problem is in the function processCampaignEvents withitn CamppaignController, specifically in the line 1113 you have code" $pending                     = $loggedCount - $logCountsProcessed;" **(Possible Fix)** and should be "$pending                     = $leadCount - $logCountsProcessed;" and for the total number instead of "$total                       = $totalYes + $totalNo;" should be "$total                       = $totalYes + $totalNo + $pending;".

### How can we reproduce this issue?

Step 1: Create contacts (more than one) with an email that you can check, create a segment that contains those contacts, create an email and create a campaign to send an email as action to that segment and a decision of "Opens email"  to trigger opened emails
Step 2: Execute the cron jobs of segments:update, campaings:update and campigns:trigger to sincronize all and send the campign emails
Step 3: Open and read for example one email of one of the contacts
Step 4: Access into mautic dashboard to Campaings, select the campaign created in the first step and check if it has the opens decision or the name you give to it in the pending statistic something > 0 and in the succes percent something different to 100%

### Relevant log output

```shell
Log doesn't track any relevant data or error related with this bug.
```

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

<br /><hr>
Care about this issue? Want to get it resolved sooner? If you are a <a href='https://www.mautic.org/become-a-member-of-mautic'>member of Mautic</a>, you can add some funds to the <a href='https://opencollective.com/mautic/projects/bounties'>Bounties Project</a> so that the person who completes this task can claim those funds once it is merged by a member of the core team! Read the docs <a href='https://contribute.mautic.org/product-team/mautic-bounty-programme'>here.</a>

**Comment by Niko25-Enting** [comment]
Hi Avara,

I hope you can help me, autoresponder emails not getting delivered, do you know how to fix this?

Thanks,

**Comment by avara** [comment]
Hi @Niko25-Enting, sorry, I'm not a Mautic expert so I can't help you with your problem because I don't know how to fix it right now. But since it is a problem that is not related to this same topic, I understand so, I recommend you write a new issue, please. That way you will receive more assistance.

**Comment by RCheesley** [comment]
Hi @avara thanks for reporting the bug and sorry for the delay in getting back to you. Does the change you've proposed fix the issue on your instance? If so would you mind making a PR on the 5.1 branch to propose fixing it in core?

Here's some docs on how to do that: https://contribute.mautic.org/contributing-to-mautic/developer/code/pull-requests.

**Comment by RCheesley** [comment]
Closing as resolved by https://github.com/mautic/mautic/pull/13915
